src/main/resources/application.properties

```diff
@@ -1,126 +1,59 @@
-spring.application.name=eiffel-intelligence
-
-# An extra optional version tag, version will be added after GitHub release version.
-# Example if build.version=1.2.3-internal becomes Version 0.0.18 (1.2.3-internal)
-build.version=
-
-# Port where Eiffel Intelligence will run
+spring.application.name: eiffel-intelligence
```

Comment:
Plesae add a comment that they should read the documentation and provide a link to the documentation section.

Reply:
Done